packages/react-native-bpk-component-image/src/BpkImage.js

```diff
@@ -56,6 +59,7 @@ const styles = StyleSheet.create({
 
 export type Props = {
   ...$Exact<ImageProps>,
+  style: ViewStyleProp,
```

Comment:
Flow was giving us an error here because `Image.style` isn't _quite_ the same as `View.style`, but we were using `Image`'s `style` type here then applying it to a view (on line 144). So, I've made it explicitly use the `style` type from `View`.